django_mri/filters/scan_filter.py

```diff
@@ -18,9 +18,7 @@ def filter_by_sequence_type(queryset, field_name, value):
         if scan.sequence_type and scan.sequence_type.id in value
     ]
     if -1 in value:
-        filtered_scan_ids += [
-            scan.id for scan in queryset if not scan.sequence_type
-        ]
+        filtered_scan_ids += [scan.id for scan in queryset if not scan.sequence_type]
```

Comment:
I'm sorry to ask but do you mind adding:
```javascript
  //// Default black line length is 88, this sets it to 79 in order to comply
  //// with flake8.
  "python.formatting.blackArgs": ["--line-length=79"],
```
to your VSC *settings.json* file? This way we can comply with flake8 and prevent formatting changes popping up everywhere.